debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/Nullable.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.sqlserver;
+
+/**
+ * 
+ * @author Jiri Pechanec
+ *
+ */
+public interface Nullable {
```

Comment:
Only a bit, I originally used it for LSN only but as I am using `NullObject` pattern for position too I thought it might get extracted into the interface.